modules/synacormediaBidAdapter.js

```diff
@@ -58,33 +64,38 @@ export const spec = {
         pos = 0;
       }
       let videoOrBannerKey = this.isVideoBid(bid) ? 'video' : 'banner';
-      getAdUnitSizes(bid).forEach((size, i) => {
-        if (!size || size.length != 2) {
-          return;
-        }
-        let size0 = size[0];
-        let size1 = size[1];
-        let imp = {
-          id: `${videoOrBannerKey.substring(0, 1)}${bid.bidId}-${size0}x${size1}`,
-          tagid: placementId
-        };
-        if (bidFloor !== null && !isNaN(bidFloor)) {
-          imp.bidfloor = bidFloor;
-        }
+      getAdUnitSizes(bid)
+        .filter(size => BLOCKED_AD_SIZES.indexOf(size.join('x')) === -1)
+        .forEach((size, i) => {
+          if (!size || size.length != 2) {
+            return;
+          }
+          let size0 = size[0];
+          let size1 = size[1];
+          let imp = {
+            id: `${videoOrBannerKey.substring(0, 1)}${bid.bidId}-${size0}x${size1}`,
+            tagid: placementId
+          };
+          if (bidFloor !== null && !isNaN(bidFloor)) {
+            imp.bidfloor = bidFloor;
+          }
 
-        let videoOrBannerValue = {
-          w: size0,
-          h: size1,
-          pos
-        };
-        if (videoOrBannerKey === 'video' && bid.params.video) {
-          Object.keys(bid.params.video)
-            .filter(param => includes(VIDEO_PARAMS, param) && !isNaN(parseInt(bid.params.video[param], 10)))
-            .forEach(param => videoOrBannerValue[param] = parseInt(bid.params.video[param], 10));
-        }
-        imp[videoOrBannerKey] = videoOrBannerValue;
-        openRtbBidRequest.imp.push(imp);
-      });
+          let videoOrBannerValue = {
```

Comment:
Since `videoOrBannerValue` does not get reassigned, I’d use `const` rather than `let`

Reply:
Turns out most of the places with `let` could actually be `const` I've updated the adapter.